driver-adapters/neon-http-vercel-nextjs/pages/api/index.js

```diff
@@ -29,6 +29,18 @@ export default async (req, res) => {
         name: true,
       },
     }),
+    create2: await prisma.user.create({
```

Comment:
```suggestion
    // Since `createMany` does not work we add a second create 
    // the record is used for `findUniqueOrThrow(...)` below
    create2: await prisma.user.create({
```